templates/style.php

```diff
@@ -386,3 +386,53 @@
 	position: absolute;
 	right: 16px;
 }
+
+<?php if ( function_exists( 'is_rtl' ) && is_rtl() ) { ?>
+
+body {
+	text-align: right;
+}
+
+blockquote {
+	border-left: 0;
+	border-right: 2px solid <?php echo sanitize_hex_color( $link_color ); ?>;
+}
+
+.amp-wp-header .amp-wp-site-icon {	
+	right: 0;
+	left: 18px;
+}
+
+.amp-wp-article-header .amp-wp-meta:last-of-type {
+	text-align: left;
```

Comment:
The changes here aren't quite right still. Instead of duplicating the rules above, the rules above should be modified like as follows:

```diff
  .amp-wp-article-header .amp-wp-meta:last-of-type {
-     text-align: right;
+     text-align: <?php echo is_rtl() ? 'left' : 'right' ?>;
  }
```

Reply:
Thanks, i do it as soon as

Reply:
hi, i did changes as you suggest in new commit, but i see there is error refer to your suggest above.